extensions/agent/src/dialogs/alertDialog.ts

```diff
@@ -113,22 +113,88 @@ export class AlertDialog extends AgentDialog<AlertData> {
+	private static readonly HTTPStorageLabel: string = localize('alertDialog.HTTPStorage', 'HTTP Storage');
+	private static readonly LatchesLabel: string = localize('alertDialog.Latches', 'Latches');
+	private static readonly LocksLabel: string = localize('alertDialog.Locks', 'Locks');
+	private static readonly LogPoolFreePoolLabel: string = localize('alertDialog.LogPoolFreePool', 'LogPool FreePool');
+	private static readonly MemoryBrokerClerksLabel: string = localize('alertDialog.MemoryBrokerClerks', 'Memory Broker Clerks');
+	private static readonly MemoryManagerLabel: string = localize('alertDialog.MemoryManager', 'Memory Manager');
+	private static readonly MemoryNodeLabel: string = localize('alertDialog.MemoryNode', 'Memory Node');
+	private static readonly PlanCacheLabel: string = localize('alertDialog.PlanCache', 'Plan Cache');
+	private static readonly QueryStoreLabel: string = localize('alertDialog.QueryStore', 'Query Store');
+	private static readonly ResourcePoolStatsLabel: string = localize('alertDialog.ResourcePoolStats', 'Resource Pool Stats');
+	private static readonly SQLErrorsLabel: string = localize('alertDialog.SQLErrors', 'SQL Errors');
+	private static readonly SQLServer2017XTPCursorsLabel: string = localize('alertDialog.SQLServer2017XTPCursors', 'SQL Server 2017 XTP Cursors');
+	private static readonly SQLServer2017XTPGarbageCollectionLabel: string = localize('alertDialog.SQLServer2017XTPGarbageCollection', 'SQL Server 2017 XTP Garbage Collection');
+	private static readonly SQLServer2017XTPIOGovernerLabel: string = localize('alertDialog.SQLServer2017XTPIOGoverner', 'SQL Server 2017 XTP IO Governer');
+	private static readonly SQLServer2017XTPPhantomProcessorLabel: string = localize('alertDialog.SQLServer2017XTPPhantomProcessor', 'SQL Server 2017 XTP Phantom Processor');
+	private static readonly SQLServer2017XTPStorageLabel: string = localize('alertDialog.SQLServer2017XTPStorage', 'SQL Server 2017 XTP Storage');
+	private static readonly SQLServer2017XTPTransactionLogLabel: string = localize('alertDialog.SQLServer2017XTPTransactionLog', 'SQL Server 2017 XTP Transaction Log');
+	private static readonly SQLServer2017XTPTransactionsLabel: string = localize('alertDialog.SQLServer2017XTPTransactions', 'SQL Server 2017 XTP Transactions');
+	private static readonly TransactionsLabel: string = localize('alertDialog.Transactions', 'Transactions');
+	private static readonly UserSettableLabel: string = localize('alertDialog.UserSettable', 'User Settable');
+	private static readonly WaitStatisticsLabel: string = localize('alertDialog.WaitStatistics', 'Wait Statistics');
+	private static readonly WorkloadGroupStats: string = localize('alertDialog.WorkloadGroupStats', 'Workload Group Stats');
+	private static readonly ObjectDropdownOptions: string[] = [AlertDialog.AccessMethodsLabel, AlertDialog.AdvancedAnalyticsLabel, AlertDialog.AvailabilityReplicaLabel,
```

Comment:
I think these should probably be queried from the server, though not sure if the list changes across different servers.  In any case, that can be changed later.